tests/integration/conftest.py

```diff
@@ -525,3 +525,25 @@ def func(project):
             time.sleep(2)
 
     return func
+
+
+@pytest.fixture
+def ontology(client):
+    ontology_builder = OntologyBuilder(
+        tools=[
+            Tool(tool=Tool.Type.BBOX, name="Box 1", color="#ff0000"),
+            Tool(tool=Tool.Type.BBOX, name="Box 2", color="#ff0000")
+        ],
+        classifications=[
+            Classification(name="Root Class",
+                           class_type=Classification.Type.RADIO,
+                           options=[
+                               Option(value="1", label="Option 1"),
+                               Option(value="2", label="Option 2")
+                           ])
+        ])
+    ontology = client.create_ontology('Integration Test Ontology',
+                                      ontology_builder.asdict(),
+                                      MediaType.Image)
+    yield ontology
+    client.delete_unused_ontology(ontology.uid)
```

Comment:
@kkim-labelbox Using a new ontology fixture that news up a new ontology for the test then marks it as deleted when it's finished. NOTE: This will a new ontology every time this fixture executes. Calling `delete_unused_ontology` sets a `deleted` flag in the db and doesn't actually remove the row. Is this okay?

Reply:
Nice fixture, yeah that should be fine for now.